sound/soc/sof/ipc4-topology.h

```diff
@@ -15,7 +15,12 @@
 #define SOF_IPC4_FW_PAGE(x) ((((x) + BIT(12) - 1) & ~(BIT(12) - 1)) >> 12)
 #define SOF_IPC4_FW_ROUNDUP(x) (((x) + BIT(6) - 1) & (~(BIT(6) - 1)))
 
-#define SOF_IPC4_MODULE_LL      BIT(5)
+#define SOF_IPC4_MODULE_LOAD_TYPE		GENMASK(3, 0)
+#define SOF_IPC4_MODULE_AUTO_START		BIT(4)
+#define SOF_IPC4_MODULE_LL		BIT(5)
+#define SOF_IPC4_MODULE_DP		BIT(6)
```

Comment:
The definitions are a bit confusing. If there are two bits for the module type (LL or DP), do we have four configurations? If we only have two, then this is somewhat broken and you need to add a comment on what values are supported.

Reply:
sure. According to spec,   LL = ~DP